frontend/packages/ceph-storage-plugin/src/models.ts

```diff
@@ -14,8 +14,8 @@ export const CephClusterModel: K8sKind = {
 };
 
 export const OCSServiceModel: K8sKind = {
-  label: 'OCS Cluster Service',
-  labelPlural: 'OCS Cluster Services',
+  label: 'Storage Cluster',
+  labelPlural: 'Storage Clusters',
```

Comment:
Why not change abbr -> SC ?

Reply:
Even, id can be changed

Reply:
SC is a storage class so make sense to have `OCS` as it differentiates. 

Id is internal and is not used to display

Reply:
@afreen23 @gnehapk ^^

Reply:
Alright, I am assuming to have Storage Cluster model be aligned to its CRD spec.
Therefore changing abbr and id makes sense .
Also, we have Portworx operator which has Storage Cluster as API and it represents SC only in console.

So are we overriding intentionally the common pattern @cloudbehl ?

Reply:
I think Abbr and id are specific to UI only and not related to CRD spec. This is used to override the default functionality. 

I'm good making it default but I found OCS more relevant & unique as an abbr. So Let me know your thoughts & I'll change it accordingly.

Reply:
Sure, for this bug fix lets have the current change in and we can discuss over other changes offline.
Thanks!